src/management/api/analytics/logs/logging-configuration.controller.ts

```diff
@@ -33,43 +38,118 @@ class ApiLoggingConfigurationController {
     private $stateParams,
     private $rootScope,
     private $scope,
-    private Constants
+    private Constants,
+    private spelGrammar: any,
   ) {
-  'ngInject';
+    'ngInject';
 
     this.initialApi = _.cloneDeep(this.$scope.$parent.apiCtrl.api);
     this.api = _.cloneDeep(this.$scope.$parent.apiCtrl.api);
-    this.initLoggingMode();
+    this.spelGrammar = spelGrammar.data;
+    this.enabled = this.api.proxy.logging && this.api.proxy.logging.mode !== 'NONE';
+    this.initLogging(this.enabled);
     this.maxDuration = Constants.org.settings.logging.maxDurationMillis;
 
     this.$scope.loggingModes = [
       {
-        name: 'None',
-        value: 'NONE'
+        title: 'Client only',
+        id: 'CLIENT',
+        icon: 'communication:shield-user',
+        description: 'to log content between the client and the gateway'
       }, {
-        name: 'Client only',
-        value: 'CLIENT'
-      }, {
-        name: 'Proxy only',
-        value: 'PROXY'
-      }, {
-        name: 'Client and proxy',
-        value: 'CLIENT_PROXY'
-      }];
+        title: 'Proxy only',
+        id: 'PROXY',
+        icon: 'communication:shield-thunder',
+        description: 'to log content between the gateway and the backend'
+      },
+      {
+        title: 'Client & proxy',
+        id: 'CLIENT_PROXY',
+        icon: 'home:earth',
+        description: 'to log content for the client and the gateway'
+      }
+    ];
+
+    this.$scope.contentModes = [
+      {
+        title: 'Headers & payloads',
+        id: 'HEADERS_PAYLOADS',
+        icon: 'finance:selected-file',
+        description: 'to log headers and payloads'
+      },
+      {
+        title: 'Headers only',
+        id: 'HEADERS',
+        icon: 'finance:file',
+        description: 'to log headers without payloads'
+      },
+      {
+        title: 'Payloads only',
+        id: 'PAYLOADS',
+        icon: 'communication:clipboard-list',
+        description: 'to log payloads without headers'
+      },
+    ];
+
+    this.$scope.scopeModes = [
+      {
+        title: 'Request & Response',
+        id: 'REQUEST_RESPONSE',
+        icon: 'navigation:exchange',
+        description: 'to log request and response'
+      },
+      {
+        title: 'Request only',
+        id: 'REQUEST',
+        icon: 'navigation:right-2',
+        description: 'to log request content only'
+      },
+      {
+        title: 'Response only',
+        id: 'RESPONSE',
+        icon: 'navigation:left-2',
+        description: 'to log response content only'
+      },
+    ];
 
     this.$scope.$on('apiChangeSuccess', (event, args) => {
       this.api = args.api;
+      this.enabled = this.api.proxy.logging.mode !== 'NONE';
     });
   }
 
+  initLogging(enabled) {
```

Comment:
```suggestion
  initLogging(enabled: boolean) {
```
related to comment on `switchMode`